index.html

```diff
@@ -44,5 +44,128 @@ <h2>I’am a Software Developer</h2>
+            <h3>Multi-Post Stories</h3>
+            <p class="Feat-text">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
+            <ul class="tech-stack-1">
+              <li>CSS</li>
+              <li>HTML</li>
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <p class="btn">See Project</p>
+          </div>
+        </div>
+      </div>
+      <div class="proj-container">
+        <div class="projects">
+          <h3>Profesional Art Printing Data</h3>
+          <p class="Feat-text">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </p>
+          <ul class="tech-stack-2">
+            <li>CSS</li>
+            <li>HTML</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <p class="btn">See Project</p>
+        </div>
+        <div class="projects">
+          <h3>Profesional Art Printing Data</h3>
+          <p class="Feat-text">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </p>
+          <ul class="tech-stack-2">
+            <li>CSS</li>
+            <li>HTML</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <p class="btn">See Project</p>
+        </div>
+        <div class="projects">
+          <h3>Profesional Art Printing Data</h3>
+          <p class="Feat-text">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </p>
+          <ul class="tech-stack-2">
+            <li>CSS</li>
+            <li>HTML</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <p class="btn">See Project</p>
+        </div>
+      </div>
+    </main>
+
+    <section class="about-section">
+      <h2>About Me</h2>
+      <p class="Feat-text">
+        Hello I’m a software developer! I can help you build a product , feature
+        or website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <sub-section class="btn">Get my Resume</sub-section>
+    </section>
+
+    <section class="skills">
+      <div class="skills-macro">
+        <div class="key-skills">
+          <img src="src/Rectangle-55.svg" alt="diamond icon" />
+          <h3>Language</h3>
+        </div>
+        <div class="skills-feat">
+          <p>Javascript</p>
+          <p>Ruby</p>
+          <p>Html</p>
+          <p>Css</p>
+        </div>
```

Comment:
- [ ] I also suggest that the technologies in each category(e.g., "JavaScript", "HTML" and "CSS") are also a list `ul` and each of them a `li` as per project requirements.